lib/uniris/oracles/oracle_server.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Uniris.Oracles.OracleCronServer do
```

Comment:
If you can change the name to `Scheduler`, to stay consistent with the other cron based modules